seed/building_sync/validation_client.py

```diff
@@ -13,16 +13,11 @@ class ValidationClientException(Exception):
 
 
 def _validation_api_post(file_, schema_version, use_case_name):
-    payload = {'schema_version': schema_version}
-    files = [
-        ('file', file_)
-    ]
-
     return requests.request(
         "POST",
         VALIDATION_API_URL,
-        data=payload,
-        files=files,
+        data={'schema_version': schema_version},
+        files={'file': (file_.name, open(file_.name, 'r').read(), 'application/xml')},
```

Comment:
We also allow folks to upload zips (and validate zips too). I didn't get the chance to test this yet, but I'm wondering if specifying this content-type will work.

Here's an example zip we use: seed/building_sync/tests/data/ex_1_and_buildingsync_ex01_measures.zip

Reply:
Ah, I had specified content type in an attempt to debug the file upload issue.  I don't think it's needed in either case however so I'll just remove it.